Detections/Cloudflare/CloudflareBadClientIp.yaml

```diff
@@ -0,0 +1,33 @@
+id: a7ce6135-9d55-4f14-b058-adc2e920a4fa
+name: Cloudflare - Bad client IP
+description: |
+  'Detects requests from IP with bad reputation index.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CloudflareDataConnector
+    dataTypes:

```

Comment:
@vu-socprime , Detection and hunting query operates on a Parser Cloudflare , dataTypes should be the Parser name and not the table name. Please update dataTypes  as Cloudflare in all files.

Reply:
updated